CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ### Fixed
 - No longer pass in `authSource: undefined` as this causes connection strings with auth to fail.
-- No longer pass in other `ssl: undefined` in connectOptions.
+- No longer pass in `ssl: undefined` in connectOptions.
+- Use `db.databaseName` when logging connection success over `config.name`. 
+- Allow Mongo Driver to infer the database being connected to over using `config.name`.
```

Comment:
This last line is not a change actually as this is the behavior in the current release: https://github.com/digitalbazaar/bedrock-mongodb/blob/d44a19830cac1d9634058a997ae790fa25929698/lib/authn.js#L148-L149